search-sentiments-map/routes/country-trends.js

```diff
@@ -37,9 +37,8 @@ router.get('/:country', (req, res) => {
 async function retrieveCountryTrends(country) {
   const query = datastore.createQuery('TrendsEntry').order('timestamp', {
     descending: true,
-  });
+  }).limit(1);
```

Comment:
Why are you adding this limit?

Reply:
It is mentioned in the Datastore documentation that as much as possible we should add limits for any datastore call. I know we only need the latest result, so I only need 1 result.

https://cloud.google.com/datastore/docs/concepts/queries